jenkins/toolbox-run.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -x
+PROJECT_ROOT="$(git rev-parse --show-toplevel)"
+
+if [ -z "$CHANGE_ID" ]; then
+    TAG="latest"
+else
+    TAG="PR-$CHANGE_ID"
+fi
+# Run one of our tests inside a newly created toolbox container
+echo "Using container image tag: $TAG"
+
+if (( $# != 2 )); then
+    >&2 cat << EOF
+    This script requires two arguments.
+
+    Usage: $0 [END_OF_TAG] [PATH_TO_SCRIPT_TO_RUN_IN_CONTAINER]
+    This script will create a toolbox container, run the script provided in it, and forcefully destroy the toolbox container.
+EOF
+    exit
+fi
+
+pushd "$PROJECT_ROOT" || exit 1
+toolbox create -y -i "quay.io/candlepin/subscription-manager:$TAG" -c "$TAG-$1"
```

Comment:
Unsure what to do here exactly. It seems the exit code is non-zero both when there are certain errors which indicate that there is no usable toolbox container (could not create etc), and when there is a toolbox container which matches the parameters given.

I think the simplest thing to do is assume that if it failed it's b/c there is already a usable container.

Reply:
Then if we cannot get a RETVAL from the execution of `toolbox run` then we know that something else went wrong rather than the tests themselves failed.